name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/OnvistaPDFExtractor.java

```diff
@@ -58,14 +58,14 @@ private void addBuyTransaction()
         });
 
         block.set(pdfTransaction);
-        pdfTransaction.section("name", "isin") //
-                        .find("Gattungsbezeichnung ISIN") //
-                        .match("(?<name>.*) (?<isin>[^ ]\\S*)$") //
+        pdfTransaction.section("name", "isin")
+                        .find("Gattungsbezeichnung ISIN")
+                        .match("(?<name>.*) (?<isin>[^ ]\\S*)$")
                         .assign((t, v) -> t.setSecurity(getOrCreateSecurity(v)))
 
-                        .section("notation", "shares") //
```

Comment:
Hast Du die leeren Kommentare mit einem bestimmten Grund entfernt?
Ich habe die immer da weil ansonsten der automatische Formatter die ganzen Ausdrücke in eine Zeile schiebt. Andrerseits schaue ich auch nicht ganz durch, wann der Formatter die schön brav auf die nächste Zeile macht.

Auf jeden Fall möchte ich jederzeit den Code per Ctrl - Shift - F formatieren können ohne mir die Ausdrücke auf eine Zeile zu hauen.